modules/statics/src/coins.ts

```diff
@@ -256,16 +261,19 @@ export const coins = CoinMap.fromCoins([
   erc20('incx', 'InternationalCryptoX', 18, '0xa984a92731c088f1ea4d53b71a2565a399f7d8d5', UnderlyingAsset.INCX),
   erc20('ind', 'Indorse', 18, '0xf8e386eda857484f5a12e4b5daa9984e06e73705', UnderlyingAsset.IND),
   erc20('inf', 'Infinitus Token', 18, '0x00e150d741eda1d49d341189cae4c08a73a49c95', UnderlyingAsset.INF),
+  erc20('inj', 'Inj', 18, '0x84bffffD702d924c6d9b25f87151bf0fb1a8913e', UnderlyingAsset.INJ),
```

Comment:
Token contract addresses in this file are typically kept in lowercase, can you remove the capital D here

Reply:
also make sure this is done in platform